Xamarin.Forms.Platform.iOS/CollectionView/ObservableItemsSource.cs

```diff
@@ -41,6 +41,14 @@ void CollectionChanged(object sender, NotifyCollectionChangedEventArgs args)
 			}
 		}
 
+		private void Replace(NotifyCollectionChangedEventArgs args)
+		{
+			var startIndex = args.NewStartingIndex > -1 ? args.NewStartingIndex : _itemsSource.IndexOf(args.NewItems[0]);
+			var count = args.NewItems.Count;
```

Comment:
This code repeats in `Add` and `Remove` methods. Perhaps it be placed in `CreateIndexesFrom` func or make another one?